[Bug]: PVC UI tests fails during StorageClass selection with selenium.common.exceptions.TimeoutException

### Description
PVC UI tests fails during StorageClass selection with following error messages:
```
ocs_ci/ocs/ui/pvc_ui.py:49: in create_pvc_ui
    self.do_click(format_locator(self.pvc_loc["storage_class_name"], sc_name))
ocs_ci/ocs/ui/base_ui.py:237: in do_click
    _do_click(locator, timeout, enable_screenshot, copy_dom)



_locator = ('//a[@id="ocs-storagecluster-ceph-rbd-link"]', 'xpath')
_timeout = 30, _enable_screenshot = False, _copy_dom = False


def _do_click(_locator, _timeout=30, _enable_screenshot=False, _copy_dom=False):
    # wait for page fully loaded only if an element was not located
    # prevents needless waiting and frequent crushes on ODF Overview page,
    # when metrics and alerts frequently updated
    if not self.get_elements(_locator):
        self.page_has_loaded()
    screenshot = (
        ocsci_config.UI_SELENIUM.get("screenshot") and enable_screenshot
    )
    if screenshot:
        self.take_screenshot()
    if _copy_dom:
        self.copy_dom()

    wait = WebDriverWait(self.driver, timeout)
    try:
        if (
            version.get_semantic_version(get_ocp_version(), True)
            <= version.VERSION_4_11
        ):
            element = wait.until(
                ec.element_to_be_clickable((locator[1], locator[0]))
            )
        else:
            element = wait.until(
                ec.visibility_of_element_located((locator[1], locator[0]))
            )
        element.click()
    except TimeoutException as e:
        self.take_screenshot()
        self.copy_dom()
        logger.error(e)

      raise TimeoutException(
            f"Failed to find the element ({locator[1]},{locator[0]})"
        )
E           selenium.common.exceptions.TimeoutException: Message: Failed to find the element (xpath,//a[@id="ocs-storagecluster-ceph-rbd-link"])
```

```
ocs_ci/ocs/ui/pvc_ui.py:49: in create_pvc_ui
    self.do_click(format_locator(self.pvc_loc["storage_class_name"], sc_name))
ocs_ci/ocs/ui/base_ui.py:237: in do_click
    _do_click(locator, timeout, enable_screenshot, copy_dom)



_locator = ('//a[@id="ocs-storagecluster-cephfs-link"]', 'xpath'), _timeout = 30
_enable_screenshot = False, _copy_dom = False


def _do_click(_locator, _timeout=30, _enable_screenshot=False, _copy_dom=False):
    # wait for page fully loaded only if an element was not located
    # prevents needless waiting and frequent crushes on ODF Overview page,
    # when metrics and alerts frequently updated
    if not self.get_elements(_locator):
        self.page_has_loaded()
    screenshot = (
        ocsci_config.UI_SELENIUM.get("screenshot") and enable_screenshot
    )
    if screenshot:
        self.take_screenshot()
    if _copy_dom:
        self.copy_dom()

    wait = WebDriverWait(self.driver, timeout)
    try:
        if (
            version.get_semantic_version(get_ocp_version(), True)
            <= version.VERSION_4_11
        ):
            element = wait.until(
                ec.element_to_be_clickable((locator[1], locator[0]))
            )
        else:
            element = wait.until(
                ec.visibility_of_element_located((locator[1], locator[0]))
            )
        element.click()
    except TimeoutException as e:
        self.take_screenshot()
        self.copy_dom()
        logger.error(e)

      raise TimeoutException(
            f"Failed to find the element ({locator[1]},{locator[0]})"
        )
E           selenium.common.exceptions.TimeoutException: Message: Failed to find the element (xpath,//a[@id="ocs-storagecluster-cephfs-link"])
```

RP link: 
https://url.corp.redhat.com/da6f47f
https://url.corp.redhat.com/b6fe2d3

--- a/ocs_ci/ocs/ui/views.py
+++ b/ocs_ci/ocs/ui/views.py
@@ -743,6 +743,10 @@ pvc_4_19 = {
     ),
 }
 
+pvc_4_20 = {
+    "storage_class_name": ('//button[@id="{}-link"]', By.XPATH),
+}
+
 storage_clients = {
     "generate_client_onboarding_ticket": (
         "//button[normalize-space()='Generate client onboarding token']",
@@ -2406,6 +2410,7 @@ locators = {
             **pvc_4_12,
             **pvc_4_14,
             **pvc_4_19,
+            **pvc_4_20,
         },
         "acm_page": {
             **acm_page_nav,

